src/Contract.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tychovbh\Mvc;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Contract extends Model
+{
```

Comment:
add class constants for available statusses and use them through project:
Define: const STATUS_CONCEPT = 'concept'

Use: Contract::STATUS_CONCEPT